swarmzero/agent.py

```diff
@@ -202,22 +204,29 @@ async def chat(
         user_id="default_user",
         session_id="default_chat",
         files: Optional[List[UploadFile]] = [],
-    ):
+        stream_mode: bool = False,
+    ) -> StreamingResponse:
```

Comment:
check what happens when stream mode false? create separate methods for chat and streaming chat